rules-presets/modify-traffic/modify-image-url.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Modify API Endpoint
+description: "Modify an image API endpoint while using Fiddler's rules."
+slug: adv_modify_image_url
+position: 20
+---
+
+# Modifying API Endpoints
+
+Apart from mapping requests to different host destinations (that return entirely new responses), you can use Fiddler to modify the ongoing traffic by changing API endpoints through their paths, or by changing and substituting parts of the HTTP requests and response headers and bodies.
+
+## Creating a "Modify Image Endpoint" Rule
+
+Create a "Modify Image Endpoint" rule by setting the following actions through the [Rules Builder]({%slug modify-traffic-get-started%}).
+
+1. Create a matching condition that uses the "When **all these conditions** are met **any number of times**" pattern. 
+
+1. Match by a **URL** that uses a string value to match the desired URL. For example, match **googlelogo** from the following URL `https://www.google.com/images/branding/googlelogo/1x/googlelogo_light_color_272x92dp.png`.
+
+1. Match by a **Method** equal to **GET**.
```

Comment:
There's a mismatch in formatting for similar steps throughout. I believe I saw it code formatted somewhere. Consider applying consistent formatting for GET throughout.